server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -315,8 +329,7 @@ public String toString() {
     private final long mgmtSrvrId = MacAddress.getMacAddress().toLong();
     private static final int ACQUIRE_GLOBAL_LOCK_TIMEOUT_FOR_COOPERATION = 5;    // 5 seconds
     private boolean _dailyOrHourly = false;
-
-    //private final GlobalLock m_capacityCheckLock = GlobalLock.getInternLock("capacity.check");
+    protected long msId = ManagementServerNode.getManagementServerId();
```

Comment:
The method  `ManagementServerNode#getManagementServerId` retrieves the MacAddress as long. To get the management ID you could use `ManagementServerHostDaoImpl#findByMsid` passing the MacAddress as long.